tests/integration/Owncloud/OcisSdkPhp/OcisTest.php

```diff
@@ -134,4 +137,58 @@ public function testCreateDriveInvalidQuota(int $quota): void
         // no new drive should have been created
         $this->assertCount($countDrivesAtStart, $ocis->listMyDrives());
     }
+
+    public function testListGroups(): void
+    {
+        $token = $this->getAccessToken('admin', 'admin');
+        $ocis = new Ocis($this->ocisUrl, $token, ['verify' => false]);
+        $groups = $ocis->listGroups();
+        $this->assertCount(SELF::GROUP_COUNT, $groups);
+        $this->assertContainsOnlyInstancesOf(Group::class, $groups);
+        foreach ($groups as $group) {
+            $this->assertIsString($group->getId());
+            $this->assertIsString($group->getDisplayName());
+            $this->assertIsArray($group->getGroupTypes());
+            $this->assertIsArray($group->getMembers());
+        }
+    }
+
+    public function testListGroupsExpanded()
+    {
+        $token = $this->getAccessToken('admin', 'admin');
+        $ocis = new Ocis($this->ocisUrl, $token, ['verify' => false]);
+        $groups = $ocis->listGroups(expandMembers: true);
+        $this->assertGreaterThan(0, count($groups));
+        foreach ($groups as $group) {
+            $this->assertIsArray($group->getMembers());
+        }
+    }
+    public function searchText()
+    {
+        return [
+            ["asdas",0,""],
+            ["user",1,"users"],
+        ];
+    }
+    public function testListGroupSerch(string $searchText, int $count,string $groupDisplatName){
```

Comment:
```suggestion
    public function testListGroupSerch(string $searchText, int $count,string $groupDisplayName){
```

I think we also need `@dataProvider searchText` in the docs